CONTRIBUTING.md

```diff
@@ -489,8 +489,7 @@ only interested in a specific component.
 - Working on a test binary? Run `(cd build && ninja bin/test_binary_name)` to
   rebuild only that test binary (without rerunning cmake). (Replace `ninja` with
   `make` if you don't have ninja installed).
-- Don't need Caffe2?  Pass `BUILD_CAFFE2_OPS=0` to disable build of
-  Caffe2 operators.
+- Don't need Caffe2?  Pass `BUILD_CAFFE2=0` to disable Caffe2 build.
```

Comment:
BUILD_CAFFE2=0 also disable other stuff. should we keep both in the doc? (IMO we can put doc optimization in a separate PR?)

Reply:
So BUILD_CAFFE2 is not a superset of BUILD_CAFFE2_OPs? If so, it should be noted in this contributing doc.

Reply:
it is actually. what I meant is to keep both lines

Reply:
e.g. 
```suggestion
- Don't need Caffe2 at all?  Pass `BUILD_CAFFE2=0` to disable Caffe2 build.
- Don't need Caffe2 ops?  Pass `BUILD_CAFFE2_OPS=0` to disable Caffe2 operator build.
```

Reply:
although I am a bit hesitate to put this on the CONTRIBUTING.md - BUILD_CAFFE2=OFF doesn't necessarily disable ALL caffe2 builds at the moment, there are still some separation work to be done. 

I wouldn't block merging the PR though.